spec/01-lexical-syntax.md

```diff
@@ -498,6 +498,65 @@ of the escape sequences [here](#escape-sequences) are interpreted.
 > [implicit conversion](06-expressions.html#implicit-conversions) from `String` to
 > `StringLike`, the method is applicable to all strings.
 
+#### Processed Strings
+
+```ebnf
+processedStringLiteral ::= alphaid ‘"’ {printableChar \ (‘"’ | ‘\$’) | escape} ‘"’ 
+                         |  alphaid ‘"""’ {[‘"’] [‘"’] char \ (‘"’ | ‘\$’) | escape} {‘"’} ‘"""’
+escape                 ::= ‘\$\$’ 
+                         | ‘\$’ letter { letter | digit } 
```

Comment:
In the implementation, it's actually `'$' (Id | this)`

see `interpolatedString` in Parsers